source/adapters/hip/usm.cpp

```diff
@@ -230,8 +230,12 @@ urUSMGetMemAllocInfo(ur_context_handle_t hContext, const void *pMem,
       }
       return ReturnValue(Pool);
     }
+    case UR_USM_ALLOC_INFO_SIZE: {
+      size_t RangeSize;
+      UR_CHECK_ERROR(hipMemPtrGetInfo(const_cast<void *>(pMem), &RangeSize));
```

Comment:
:)

Reply:
Yeah.. I don't feel too bad as it's done in quite a few other places.

Reply:
It's on HIP, great API design.